spec/models/user_spec.rb

```diff
@@ -9,13 +9,35 @@
   end
   subject { build_stubbed :user }
 
-  it { should have_one :subscription }
+  context "included modules" do
+    it { expect(User.ancestors.include?(Filterable)).to eq(true) }
```

Comment:
this reads a little strange - maybe we can't do anything about it, but could we have:

```rb
it 'includes Filterable module' do
  expect(User.ancestors).to include(Filterable)
end